util/options.d.ts

```diff
@@ -68,3 +68,6 @@ export function resolvePath(path: string, baseDir: string, useNodeResolution?: b
 
 /** Populates default values on a parsed options result. */
 export function addDefaults(config: Config, options: OptionSet): void;
+
+/** Sanitizes an option value to be a valid value of the option's type. */
+export function sanitizeValue(value: any, type: string): boolean | number | string | null | number[] | string[]; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
unrelated change

Reply:
Hm, this is only necessary because this function is tested directly...is there a way to avoid that?

Reply:
Could you test this case by using `merge` API?